src/amber/controller/helpers/tags.cr

```diff
@@ -0,0 +1,7 @@
+module Amber::Controller::Helpers
```

Comment:
Perhaps, although it is directly related to controllers and about as useful as redirect or render. I think I'll merge it for now and when we start adding form helpers and stuff we can revisit this issue.